packages/syft/src/syft/core/tensor/tensor.py

```diff
@@ -351,14 +348,14 @@ def __init__(
     ) -> None:
         """data must be a list of numpy array"""
 
-        if isinstance(child, (list, np.int32)):
-            child = to32bit(np.array(child), verbose=False)
+        if isinstance(child, list) or np.isscalar(child):
+            child = np.array(child)
```

Comment:
Q: Another idea, should we force here np 64 bit?

Reply:
Shall we assign the default numpy type in config?